src/nashpy/egt/imitation_dynamics.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from typing import Generator, Tuple, Any
+import numpy.typing as npt
+
+def payoff(player_strategy, opponent_strategy, player_payoff_matrix):
+    '''
```

Comment:
```suggestion
    """
```

Reply:
Change these throughout.

Reply:
Removed the main() calls from the test files and updated all code to use black formatting.